src/Model/EnumTypeGo.cs

```diff
@@ -43,13 +50,50 @@ public string GetEmptyCheck(string valueReference, bool asEmpty)
                 Values
                     .ForEach(v =>
                     {
-                        constants.Add(HasUniqueNames ? v.Name : Name + v.Name, v.SerializedName);
+                        constants.Add(FormatValue(v.Name), v.SerializedName);
                     });
 
                 return constants;
             }
         }
 
         public string Documentation { get; set; }
+
+        private string FormatName(string rawName)

```

Comment:
There are already some funcs that do symilar stuff in the namer. I would like to avoid confusion